### PR TITLE
Update fields instead saving object

### PR DESCRIPTION
**Pull request by Boquete**
I want to merge this change because we should not save Order. 
### Pull Request Checklist

1. [ ] Privileged views and APIs are guarded by proper permission checks.
1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] Database queries are optimized and the number of queries is constant.
1. [ ] Database migration files are up to date.
1. [ ] The changes are tested.
1. [ ] GraphQL schema and type definitions are up to date.
1. [ ] Changes are mentioned in the changelog.

### CODE REVIEW

File: saleor/order/utils.py [hunk 1]
```diff
@@ -99,7 +99,7 @@ def recalculate_order(order: Order, **kwargs):
     if order.discount:
         total -= order.discount
     order.total = total
-    order.save()
+    order.save(update_fields=["discount_amount", "total"])
```
Comment (NyanKiyoshi) [inline saleor/order/utils.py]: `total` is not a field, but multiple ones:

```suggestion
    order.save(update_fields=["discount_amount", "total_net_amount", "total_gross_amount", "currency"])
```

File: saleor/order/utils.py [hunk 2]
```diff
@@ -138,7 +138,7 @@ def update_order_prices(order, discounts):
 
     if order.shipping_method:
         order.shipping_price = manager.calculate_order_shipping(order)
-        order.save()
+        order.save(update_fields=["shipping_price"])
```
Comment (NyanKiyoshi) [inline saleor/order/utils.py]: Same for `shipping_price`:
```suggestion
        order.save(update_fields=["shipping_price_net_amount", "shipping_price_gross_amount", "currency"])
```

**Comment by django-queries** [comment]
Here is the report for 1150d6525a26c89d552b65173e35b315d96863ae (Boquete/saleor @ patch-1)
Base comparison is 7ab9e38e7785f1281eabfd47b970e9469ecf3195.



<details><summary>No differences were found. (click me)</summary>
<p>

```diff
# api.benchmark checkout
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  add billing address to checkout            	         34	         34	             20
  add shipping to checkout                   	          7	          7	              0
  checkout payment charge                    	         14	         14	              0
  complete checkout                          	          6	          6	              0
  create checkout                            	         48	         48	             24

# api.benchmark homepage
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  retrieve main menu                         	          5	          5	              0
  retrieve product list                      	          4	          4	              0
  retrieve secondary menu                    	          5	          5	              0
  retrieve shop                              	          2	          2	              0

# api.benchmark product
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  product details                            	         15	         15	              3
  retrieve product attributes                	         13	         13	              2

# api.benchmark variant
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  product variant bulk create                	         51	         51	              3
  retrieve variant list                      	         15	         15	              6

# api product sorting attributes
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  sort product not having attribute data     	         21	         21	              0
```

</p>
</details>

**Comment by maarcingebala** [comment]
@Boquete Final call to become a Saleor contributor 😉. If no further activity occurs here, we'll open a PR with this code, since this is a useful change.

**Comment by Boquete** [comment]
@maarcingebala please do, I'm afk for week so can't make changes rn

**Comment by maarcingebala** [comment]
@Boquete Alright, thanks for the update.

**Comment by maarcingebala** [comment]
Fixed in #4976

**Pull request closed without merging**